kong/templates/nginx_kong_gui_include.lua

```diff
@@ -93,5 +93,10 @@ location ~* ^$(admin_gui_path_prefix)(?<path>/.*)?$ {
 > end
     sub_filter_once off;
     sub_filter_types *;
+
+    log_by_lua_block {
+        ngx.ctx.km_visit_mark = true
```

Comment:
There is no need to add `ngx.ctx.km_visit_mark` flag? Nginx can make sure  `log_by_lua_block` only executes once per HTTP request.